src/components/TeamSection.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <section class="team-section">
+    <div class="container">
+      <h2 class="text-center font-weight-bold pt-5 pb-3">{{ displayheading("OUR TEAM").first }}<span class="highlighted">{{ displayheading("OUR TEAM").second }}</span>{{ displayheading("OUR TEAM").third }}</h2>
+      <carousel autoplay-timeout="5000" :navigation-enabled="true" :loop="true" :autoplay="true" :pagination-enabled="false" :per-page-custom="[[280,1],[768, 3]]" :navigation-next-label="`<i class=&quot;fas fa-chevron-circle-right fa-2x d-none d-md-block&quot; aria-hidden=&quot;true&quot;></i>`" :navigation-prev-label="`<i class=&quot;fas fa-chevron-circle-left fa-2x d-none d-md-block&quot; aria-hidden=&quot;true&quot; />`" :autoplay-hover-pause="true" class="container p-0">
+        <slide v-for="(item,index) in profiles" :key="index" class="col-md-4 col-sm-12 p-2">
+          <div class="card  rounded-0">
+            <img class="card-img-top rounded-0" :src="item.image" alt="Card image cap">
+            <div class="overlay-image" />
+            <div class="card-body">
+              <h5 class="card-title font-weight-bold">{{ displayheading(item.first_name+" "+item.last_name).first }}<span class="highlighted">{{ displayheading(item.first_name+" "+item.last_name).second }}</span>{{ displayheading(item.first_name+" "+item.last_name).third }}</h5>
+              <h6 class="card-subtitle mb-2 text-muted">{{ item.alias }}</h6>
+              <div class="icon-block text-center">
+                <a :href="item.facebook_link"><i class="fab fa-facebook-f" /></a>
+                <a :href="item.git_link"><i class="fab fa-github" /></a>
+                <a :href="item.email"><i class="fab fa-google-plus-g" /></a>
+              </div>
+            </div>
+          </div>
+        </slide>
+      </carousel>
+      <p class="highlighted text-right m-0 mt-2 mr-2"><u><a> Meet all the members »</a></u></p>
+    </div>
+  </section>
+</template>
+
+<script>
+import { Carousel, Slide } from 'vue-carousel'
+import Members from '@/services/members.js'
+
+export default {
+  components: {
+    Carousel,
+    Slide
+  },
+  data () {
+    return {
+      profiles: []
+    }
+  },
+
+  async created () {
+    try {
+      const team = (await Members.getMembers()).data
+      console.log(team)
+      team.forEach((mem) => {
+        this.categorise(mem)
+      })
+    } catch (e) {
+      console.log(e.message)
+    }
+  },
+  methods: {
+    categorise (mem) {
+      if (mem.degree_name === 'BTECH' & mem.year_name === 4) {
+        this.profiles.push(mem)
+      }
+    },
+    displayheading (text) {
+      var length = text.length
+      var middle = length / 2
+      var highlighted = 3
+      return {
+        first: text.substr(0, middle),
+        second: text.substr(middle, highlighted),
+        third: text.substr(highlighted + middle, length)
+      }
+    }
+  }
+}
+</script>
+
+<style>
```

Comment:
Your style should be scoped. If scoped styling isn't working for some comments, add proper class name to it so that it is easiliy recognizable and but the style in the global style file `static/css/global.css`.